src/components/EventList/components/EventItem/EventItem.module.scss

```diff
@@ -9,25 +9,25 @@
 }
 
 .event_item:hover {
-  .event_column, .descriptionText {
-  color: $neutral-white;
+  .event_column,
+  .descriptionText {
+    color: var(--mui-palette-primary-contrastText);
   }
 }
 
 :global(.gc360_event_list) > section {
-  background-color: $neutral-white;
+  background-color: var(--mui-palette-primary-contrastText);
```

Comment:
This should probably be neutral light, since it's a background color, and not text(I assume)